wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -148,7 +148,7 @@ void deadlineWithTest() {
       Command endsBefore = new InstantCommand();
       Command endsAfter = new WaitUntilCommand(() -> false);
 
-      Command group = dictator.deadlineWith(endsBefore, endsAfter);
+      Command group = dictator.deadlineFor(endsBefore, endsAfter);
```

Comment:
Is there no equivalent C++ test?

Reply:
It seems like it- Compared to Java, C++'s `CommandDecoratorTest.cpp` file does not have tests for `AndThen(cmd)`\*, `DeadlineWith()`, `AlongWith()`, and `RaceWith()`. It looks like the Java versions of those tests were there since the beginning (558c383), while the C++ version never had them (076ed77 didn't have them, at least, despite the command classes for those classes being added at the same time).
\*Java has two tests for `andThen()`, one for `andThen(() -> actionToDoOnce(), requirements)` and one for `andThen(cmd)`. C++ only has the `AndThen(cmd)` test.

Reply:
Most decorators for C++ were added much later when CommandPtr was introduced (2023).